package.json

```diff
@@ -27,8 +27,9 @@
     "prepare-preview-builds": "./scripts/prepare-preview-builds.sh",
     "publish-previews": "yarn workspaces foreach --parallel --verbose run publish:preview",
     "setup": "yarn install",
-    "test": "yarn workspaces foreach --parallel --verbose run test",
-    "test:clean": "yarn workspaces foreach --parallel --verbose run test:clean && yarn test"
+    "test": "yarn test:verbose --collectCoverage=false --reporters=jest-silent-reporter",
```

Comment:
What do you think about moving these flags down a level so that we always get the same behavior whether we run this script at the root level or package level, and we don't have to keep these flags synchronized?

``` suggestion
    "test": "yarn workspaces foreach --parallel --verbose run test",
```

Reply:
The idea here is to disable coverage reports at root level and enable them at package level. At the root level, they contribute to making the output unreadable, but the package level scripts are used in CI, so we need the coverage threshold to be checked.

Reply:
Okay. I'm considering a change where coverage reports are removed by default even on the package level (because I run individual files a lot and I always have to pass `--no-coverage --verbose=false`), but that can come in a different PR. Your explanation makes sense.

Reply:
I could add a `test:silent` or `test:no-coverage" package-level script for that here, unless there are related changes you'd want to collect in a separate PR?